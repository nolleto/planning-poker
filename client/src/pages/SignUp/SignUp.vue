<template>
  <div :class="$style.wrapper">
    <sign-up-form :class="$style.form" />
  </div>
</template>

<script>
import SignUpForm from '../../containers/SignUpForm'

export default {
  name: 'SignUp',

  components: { SignUpForm }
}
</script>

<style module>
  .wrapper {
    display: flex;
    align-items: center;
    justify-content: center;
  }

  .form {
    width: 100%;
    max-width: 600px;
    padding: 20px;
  }
</style>
