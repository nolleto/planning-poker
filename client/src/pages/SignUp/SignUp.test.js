import SignUp from './index'
import { shallowMount } from '@vue/test-utils'

const shallowComponent = () => {
  return shallowMount(SignUp)
}

describe('SignUp component', () => {
  let wrapper

  describe('Snapshots', () => {
    beforeAll(() => {
      wrapper = shallowComponent()
    })

    it('matches', () => {
      expect(wrapper.element).toMatchSnapshot()
    })
  })
})
