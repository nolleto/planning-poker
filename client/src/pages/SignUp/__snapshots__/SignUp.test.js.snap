// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignUp component Snapshots matches 1`] = `<signupform-stub />`;
